tests/functional/raw-node/utils/gcpUtils.js

```diff
@@ -1,15 +1,8 @@
 const { makeGcpRequest } = require('./makeRequest');
 
 function gcpRequestRetry(params, retry, callback) {
-    const retryTimeout = {
-        0: 0,
-        1: 1000,
-        2: 2000,
-        3: 4000,
-        4: 8000,
-    };
     const maxRetries = 4;
-    const timeout = retryTimeout[retry];
+    const timeout = Math.pow(2, retry) * 1000;
```

Comment:
use of `pow` scares me haha, why the exponential?

Reply:
It is an implementation of [exponential backoff](https://cloud.google.com/storage/docs/exponential-backoff) to perform retries after an increasing wait time.